data/nlu.yml

```diff
@@ -99,6 +113,19 @@ nlu:
     - [escalated]{"entity": "priority", "value": "high"}
     - [escalate]{"entity": "priority", "value": "high"}
     - [super high](priority)
+    - anna.steinberg@tech4germany.org
```

Comment:
I think we shouldn't use none rasa emails or change the names on these, wyt @b-quachtran ?

Reply:
Ah yeah, PII. That makes sense to remove.